src/components/inventory/expand.js

```diff
@@ -1,9 +1,34 @@
 import React from 'react'
-import { Card, Carousel } from 'antd'
-
+import { Card, Carousel, Modal, message } from 'antd'
+import AxiosAuth from '../Auth/axiosWithAuth'
+import { useDispatch } from 'react-redux'
+import * as creators from '../../state/actionCreators'
+const { confirm } = Modal
 const { Meta } = Card
 
 const Expanded = ({ inventory }) => {
+  const dispatch = useDispatch()
+  function showDeleteConfirm (id) {
+    confirm({
+      title: 'Are you sure you want to delete this item?',
+      okText: 'Yes',
+      okType: 'danger',
+      cancelText: 'No',
+      onOk () {
+        AxiosAuth().delete(`https://shopping-cart-eu3-staging.herokuapp.com/api/store/products/${id}`)
+          .then(res => {
+            dispatch(creators.getCurrentUser())
+            message.success('item deleted')
```

Comment:
Assuming this message is displayed to the user, can we capitalise it?

Reply:
@shaunorpen I'll capitalize first letter of each word, to remain consistent with rest of the codebase. Is that okay? or you think we should change everything to all caps?

Reply:
Just capitalise to maintain consistency I think. Cheers!

Reply:
@shaunorpen done

Reply:
Looks Good